src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -830,6 +842,24 @@ internal IEnumerable<SecurityKey> GetContentEncryptionKeys(JsonWebToken jwtToken
             {
                 try
                 {
+#if NET472
```

Comment:
We need to add a CORE 3.1 target

Reply:
Added in [f6f8794](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1834/commits/f6f8794f6309d9962e52064dd47858458ee10130)